utils/misc/dbutils.go

```diff
@@ -11,13 +11,16 @@ import (
 )
 
 // GetConnectionString Returns Jobs DB connection configuration
-func GetConnectionString() string {
-	host := config.GetString("DB.host", "localhost")
-	user := config.GetString("DB.user", "ubuntu")
-	dbname := config.GetString("DB.name", "ubuntu")
-	port := config.GetInt("DB.port", 5432)
-	password := config.GetString("DB.password", "ubuntu") // Reading secrets from
-	sslmode := config.GetString("DB.sslMode", "disable")
+func GetConnectionString(c *config.Config) string {
+	if c == nil {
+		c = config.Default
+	}
```

Comment:
Passing a config shouldn't be optional, in the long run we would like to get rid of `config.Default`, let's not introduce it where we can avoid it :)

Reply:
a lot of dependants of this function don't have specific config, would need more refactoring in various areas of the repo.

Reply:
Yes as simple as that... just present the naked truth in plain sight!